src/V3Error.h

```diff
@@ -48,6 +48,7 @@ class V3ErrorCode final {
         // Boolean information we track per-line, but aren't errors
         I_CELLDEFINE,   // Inside cell define from `celldefine/`endcelldefine
         I_COVERAGE,     // Coverage is on/off from /*verilator coverage_on/off*/
+        I_OPT_DFG,      // DFG optimizer is on/off from /*verilator opt_dfg_on/off*/
```

Comment:
This is for debugging, to make it possible to bisect the input (via the .vlt config file) to locate logic that is mis-optimized (it's pretty hopeless to track it down otherwise in big designs). Do you have an alternative suggestion that would allow for that without this mechanism?

Also used by t_dfg_peephole for testing but I can change that.

BTW I agree this should not be something the user should have to use, unless asked explicitly for debugging, so should be considered a "hidden" option in any case.

Reply:
Removed and rewritten tests that used this.